models/User.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const userSchema = new Schema({
+  googleId: String,
```

Comment:
so youre storing the `googleId` in the database, doesn't it change at some point ? and is there a reason to store it in the database?

Reply:
It doesn't change. And I need to store it the database so as to compare it with what Google sends anytime a user tries to log in. It's the one unique thing I get when a user logs in or signs up.

Reply:
but why do you need to check what `Google` has sent you back ?

Reply:
To see if I have that user so that I can authenticate them without making several requests. 
I store my users if they exist I don't go through the entire process again. For a new user I have to store and authenticate

Reply:
nice love the idea